PROVESFlightControllerReference/test/int/rtc_test.py

```
@@ -436,3 +436,20 @@ def test_10_double_set_test(fprime_test_api: IntegrationTestAPI, start_gds):
     fprime_test_api.send_command(f"{rtcManager}.ALARM_SET", [alarm_time_data_str])
     # Assert that we receive an AlarmNotSet event within 10 seconds
     fprime_test_api.await_event(f"{rtcManager}.AlarmNotSet", timeout=10)
+
+
+@pytest.mark.uart_only(reason="Test functionality of to_proc_time")
+def test_proc_toggle(fprime_test_api: IntegrationTestAPI, start_gds):
```

Ruff (0.15.21)

[warning] Unused function argument: `start_gds`

(ARG001)